deploy/packages/bbp-packages.yaml

```diff
@@ -173,6 +173,7 @@ packages:
       - mpi
       - python
     specs:
+      - neurodamus-core
```

Comment:
to clarify : Is this +common or ~common?  (on line 154 we are specifying variant)

Reply:
`~common`